### PR TITLE
Sentry fixes

### PR DESCRIPTION
**Pull request by zotya**

### CODE REVIEW

**Review by tisto — APPROVED** [review]

File: src/components/theme/App/App.jsx [hunk 1]
```diff
@@ -83,6 +85,14 @@ class App extends Component {
    */
   componentDidCatch(error, info) {
     this.setState({ hasError: true, error, errorInfo: info });
+    if (__CLIENT__) {
+      if (
+        (window.env && window.env.RAZZLE_SENTRY_DSN) ||
+        (__SENTRY__ && __SENTRY__.SENTRY_DSN)
```
Comment (nzambello) [inline src/components/theme/App/App.jsx:91]:
```suggestion
        (window?.env?.RAZZLE_SENTRY_DSN || __SENTRY__?.SENTRY_DSN)
```

**Review by nzambello — APPROVED** [review]
LGTM.
I only added a suggestion

**Review by avoinea — APPROVED** [review]

File: src/components/theme/App/App.jsx (hunk 1 above)
Comment (zotya) [reply src/components/theme/App/App.jsx]: thanks for the hint

**Comment by avoinea** [comment]
@tisto Can we have another alpha release with this merge, pls?

**Comment by tisto** [comment]
@avoinea sure. Will merge and release as soon as gh actions are green.

**Comment by tisto** [comment]
@avoinea @zotya here you go: https://www.npmjs.com/package/@plone/volto/v/8.5.0-alpha.1

**Comment by tisto** [comment]
@avoinea @zotya @nzambello can you give me a heads up how this works in prod? I am just wondering if we can release a final soonish or if we have to branch Volto to continue to move fast. PRs are piling up. :)

**Comment by avoinea** [comment]
@tisto LGTM

Already deployed it on staging:

**entrypoint**: 
- https://github.com/eea/ims-frontend/blob/64d0cdec627e7cf3c531d429258c735810ca5b18/entrypoint-prod.sh#L25

**docker-compose:** 
- https://github.com/eea/eea.rancher.catalog/blob/c9f7444a11963aeeb908d0e77f6dbd59180e8ab9/templates/volto-ims/32/docker-compose.yml#L14-L20

**Comment by avoinea** [comment]
@tisto @zotya @sneridagh @tiberiuichim @nzambello  

Maybe we should move [runtime_config.js](https://github.com/plone/volto/blob/master/src/runtime_config.js) to helpers or somewhere else in order to be able to easily override it via customization / addons?!

**Comment by avoinea** [comment]
Ok. No need to move it, just to fix the import. PR added https://github.com/plone/volto/pull/1922

**Pull request merged**